### DISCUSSION TITLE
Integration GrapesJS with KeystoneJS

### DISCUSSION THREAD

**Discussion by Raj-Gohel** [keystonejs/keystone#7975] [Questions]
I am searching for possible ways to integrate [GrapesJS](https://grapesjs.com/) with KeystoneJS.

I tried this https://www.npmjs.com/package/keystonejs-grapesjs-editor but it does not seem to work. NPM installation for this package is stuck in this case.

Is there any other way to integrate the same?

**Comment by flexdinesh** [comment]
Looks like https://www.npmjs.com/package/keystonejs-grapesjs-editor was built for keystone 5 (older version). We're not sure if or how it will work with keystone 6. But from a quick look at GraphJS it looks awfully similar to Keystone 6 document field's [custom components](https://keystonejs.com/docs/guides/document-fields#component-blocks).
